services/graph/pkg/identity/ldap.go

```diff
@@ -320,7 +384,7 @@ func (i *LDAP) getEntryByDN(dn string, attrs []string, filter string) (*ldap.Ent
 		Msg("getEntryByDN")
 	res, err := i.conn.Search(searchRequest)
 	if err != nil {
-		i.logger.Error().Err(err).Str("backend", "ldap").Str("dn", dn).Msg("Search user by DN failed")
+		i.logger.Error().Err(err).Str("backend", "ldap").Str("dn", dn).Msg("Search ldap by DN failed")
```

Comment:
maybe make it `search ldap for user by DN`.

Reply:
No. I changed it to "ldap" because this is also used for finding the group.